<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemEditorDialog</class>
 <widget class="QDialog" name="ItemEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeMe!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabMain">
      <attribute name="title">
       <string>&amp;Основная информация</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="8" column="0">
        <widget class="QLabel" name="lblEndDate">
         <property name="text">
          <string>Дата закрытия</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="CDateEdit" name="edtEndDate">
         <property name="displayFormat">
          <string>dd.MM.yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblCode">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Код</string>
         </property>
         <property name="buddy">
          <cstring>edtCode</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Наименование</string>
         </property>
         <property name="buddy">
          <cstring>edtName</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblRegionalCode">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Региональный код</string>
         </property>
         <property name="buddy">
          <cstring>edtRegionalCode</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblService">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Базовая &amp;услуга</string>
         </property>
         <property name="buddy">
          <cstring>cmbService</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblMedicalAidProfile">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Профиль медицинской помощи</string>
         </property>
         <property name="buddy">
          <cstring>cmbMedicalAidProfile</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblEmsProfileCode">
         <property name="text">
          <string>Код профиля в бюро госпитализации</string>
         </property>
         <property name="buddy">
          <cstring>edtEmsProfileCode</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lblUsishCode">
         <property name="text">
          <string>Код ЕГИСЗ</string>
         </property>
         <property name="buddy">
          <cstring>edtUsishCode</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lblTfomsCode">
         <property name="text">
          <string>Код ТФОМС</string>
         </property>
         <property name="buddy">
          <cstring>edtTfomsCode</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>307</width>
           <height>63</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="1" colspan="2">
        <widget class="QLineEdit" name="edtTfomsCode"/>
       </item>
       <item row="6" column="1" colspan="2">
        <widget class="QLineEdit" name="edtUsishCode"/>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QLineEdit" name="edtEmsProfileCode"/>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="CRBComboBox" name="cmbMedicalAidProfile"/>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="CRBComboBox" name="cmbService"/>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="edtRegionalCode"/>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="edtName"/>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="edtCode"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabIdentification">
      <attribute name="title">
       <string>&amp;Идентификация</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="CInDocTableView" name="tblIdentification"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>edtCode</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>edtRegionalCode</tabstop>
  <tabstop>cmbService</tabstop>
  <tabstop>cmbMedicalAidProfile</tabstop>
  <tabstop>edtEmsProfileCode</tabstop>
  <tabstop>edtUsishCode</tabstop>
  <tabstop>edtTfomsCode</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>tblIdentification</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
